docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -198,43 +198,192 @@ This table can then be queried in Trino::
 
     SELECT * FROM cassandra.mykeyspace.users;
 
-Data types
-----------
-
-The data types mappings are as follows:
-
-================  ======
-Cassandra         Trino
-================  ======
-ASCII             VARCHAR
-BIGINT            BIGINT
-BLOB              VARBINARY
-BOOLEAN           BOOLEAN
-DATE              DATE
-DECIMAL           DOUBLE
-DOUBLE            DOUBLE
-FLOAT             REAL
-INET              IPADDRESS
-INT               INTEGER
-LIST<?>           VARCHAR
-MAP<?, ?>         VARCHAR
-SET<?>            VARCHAR
-SMALLINT          SMALLINT
-TEXT              VARCHAR
-TIMESTAMP         TIMESTAMP(3) WITH TIME ZONE
-TIMEUUID          UUID
-TINYINT           TINYINT
-TUPLE             ROW with anonymous fields
-UUID              UUID
-UDT               ROW with field names
-VARCHAR           VARCHAR
-VARINT            VARCHAR
-================  ======
-
-Any collection (LIST/MAP/SET) can be designated as FROZEN, and the value is
-mapped to VARCHAR. Additionally, blobs have the limitation that they cannot be empty.
-
-Types not mentioned in the table above are not supported.
+.. _cassandra-type-mapping:
+
+Type mapping
```

Comment:
Could you reorder entries in both read and write mapping as https://trino.io/docs/current/language/types.html?

Reply:
@ebyhr did my best, please review again. The original table was alpha order.